recipes-mender/mender/files/mender.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Mender OTA update service
+
+[Service]
+Type=simple
+User=root
+Group=root
+ExecStart=/usr/bin/mender
```

Comment:
I envision we might call mender with something like `--daemon` here. But at the moment we don't have it.